packages/xchain-thorchain/__tests__/client.test.ts

```diff
@@ -104,10 +104,23 @@ describe('Client Test', () => {
   const mainnet_address_path1 = 'thor1hrf34g3lxwvpk7gjte0xvahf3txnq8ecgaf4nc'
   const testnet_address_path0 = 'tthor19kacmmyuf2ysyvq3t9nrl9495l5cvktj5c4eh4'
   const testnet_address_path1 = 'tthor1hrf34g3lxwvpk7gjte0xvahf3txnq8ecv2c92a'
-
+  const clientUrl = {
+    [Network.Testnet]: {
+      node: 'https://thornode.ninerealms.com',
+      rpc: 'https://rpc.ninerealms.com',
```

Comment:
Are tests for `testnet` still working by pointing to `mainnet` urls? If yes and because of mocking server requests, you could can leave those urls as empty string to avoid any confusion.